install.sh

```diff
@@ -32,11 +32,28 @@ if [ -e "$HOME/$BACKUP_FILE" ]; then
     done
 fi
 
-test -w "$HOME/$CONFIG_FILE" &&
-  cp -a "$HOME/$CONFIG_FILE" "$HOME/$CONFIG_FILE.bak" &&
-  echo -e "\033[0;32mYour original $CONFIG_FILE has been backed up to $CONFIG_FILE.bak\033[0m"
+while true
+do
+  read -e -n 1 -r -p "Would you like to keep your $CONFIG_FILE and append bash-it templates at the end? [y/N] " choice
+    case $choice in
+      [yY])
+	    (sed "s|{{BASH_IT}}|$BASH_IT|" "$BASH_IT/template/bash_profile.template.bash" || tail -n +2) >> "$HOME/$CONFIG_FILE"
+	    echo -e "\033[0;32mBash-it template has been added to your $CONFIG_FILE\033[0m"
+	    break
+        ;;
+      [nN]|"")
+	    test -w "$HOME/$CONFIG_FILE" &&
+	    cp -aL "$HOME/$CONFIG_FILE" "$HOME/$CONFIG_FILE.bak" &&
```

Comment:
I think we should back up the original file regardless of what we're doing. The reason is that the `uninstall.sh` script is expecting to find a backup file when the user is trying to uninstall Bash-it.

I suggest you copy these three lines (`test` && `cp` && echo`) up to the first case before you append the template content to the user's config file.